docs/getting-started-kubernetes.rst

```diff
@@ -857,19 +857,87 @@ The ``pod.yaml`` configuration file for such a deployment:
 
-To apply SriovNetworkNodePolicy on several nodes in parallel, specify the ``maxParallelConfiguration`` option in the SriovOperatorConfig CRD:
+To apply configure SR-IOV on several nodes in parallel, the ``SriovNetworkPoolConfig`` CRD and specify number or percent of unavailable nodes at the moment:
 
+``sriov-network-pool-config-number.yaml``
+
+.. code-block:: yaml
+
+    apiVersion: v1
+    kind: SriovNetworkPoolConfig
+    metadata:
+      name: pool-1
+      namespace: network-operator
+    spec:
+      maxUnavailable: "20"
+      nodeSelector:
+        - matchExpressions:
+          - key: some-label
+            operator: In
+            values:
+              - val-2
+        - matchExpressions:
+          - key: other-label
+            operator: "Exists"
+
+``sriov-network-pool-config-percent.yaml``
+
+.. code-block:: yaml
+
+    apiVersion: v1
+    kind: SriovNetworkPoolConfig
+    metadata:
+      name: pool-1
+      namespace: network-operator
+    spec:
+      maxUnavailable: "10%"
+      nodeSelector:
+        - matchExpressions:
+          - key: some-label
+            operator: In
+            values:
+              - val-2
+        - matchExpressions:
+          - key: other-label
+            operator: "Exists"
+
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Upgrade from NVIDIA Network Operator v24.1.0
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To upgrade SR-IOV Network operator you need to create ``SriovNetworkPoolConfig`` CRD with the number of nodes to be configured in a parallel as we did in `SriovOperatorConfig`` in previous releases.
+
+E.g.: old method to congigure nodes in a prarallel:
 .. code-block:: bash
 
-    kubectl patch sriovoperatorconfigs.sriovnetwork.openshift.io -n network-operator default --patch '{ "spec": { "maxParallelNodeConfiguration": 0 } }' --type='merge'
+    kubectl patch sriovoperatorconfigs.sriovnetwork.openshift.io -n network-operator default --patch '{ "spec": { "maxParallelNodeConfiguration": 5 } }' --type='merge'
+
+
+New  method to congigure nodes in a prarallel:
+
+``sriov-network-pool-config-new.yaml``
+
+.. code-block:: yaml
+
+    apiVersion: v1
+    kind: SriovNetworkPoolConfig
+    metadata:
+      name: pool-1
+      namespace: network-operator
+    spec:
+      maxUnavailable: "5"
+      nodeSelector:
+        - matchExpressions:
+          - key: node-role.kubernetes.io/master
+                operator: Exists
 
 --------------------------------------------------------------------------
 SR-IOV Network Operator Deployment – Parallel NIC Configuration for SR-IOV
 --------------------------------------------------------------------------
 
-.. warning:: This is a Tech Preview feature, which is supported only for Vanilla Kubernetes deployments with SR-IOV Network Operator.
+.. warning:: This feature, which is supported only for Vanilla Kubernetes deployments with SR-IOV Network Operator.
```

Comment:
suggestion:
This feature is supported only for Vanilla Kubernetes

Reply:
done